mix.lock

```diff
@@ -50,12 +50,12 @@
   "membrane_opus_format": {:hex, :membrane_opus_format, "0.3.0", "3804d9916058b7cfa2baa0131a644d8186198d64f52d592ae09e0942513cb4c2", [:mix], [], "hexpm", "8fc89c97be50de23ded15f2050fe603dcce732566fe6fdd15a2de01cb6b81afe"},
   "membrane_raw_video_format": {:hex, :membrane_raw_video_format, "0.2.0", "cda8eb207cf65c93690a19001aba3edbb2ba5d22abc8068a1f6a785ba871e8cf", [:mix], [], "hexpm", "6b716fc24f60834323637c95aaaa0f99be23fcc6a84a21af70195ef50185b634"},
-  "membrane_rtc_engine": {:git, "https://github.com/jellyfish-dev/membrane_rtc_engine.git", "c4c94c70cdeeac620726fa9230fcd6e3820ef75a", []},
+  "membrane_rtc_engine": {:git, "https://github.com/jellyfish-dev/membrane_rtc_engine.git", "c7c720e3f49e4357779e71cb657ef4ce11cffbe4", []},
   "membrane_rtc_engine_timescaledb": {:hex, :membrane_rtc_engine_timescaledb, "0.1.0", "d0b5657a5555e9b6a85a0327ce9f95c5d9b7f4e2e6b2d1d1527be8e51852cd17", [:mix], [{:ecto_sql, "~> 3.7", [hex: :ecto_sql, repo: "hexpm", optional: false]}, {:membrane_rtc_engine, "~> 0.8.0", [hex: :membrane_rtc_engine, repo: "hexpm", optional: false]}, {:postgrex, "~> 0.16", [hex: :postgrex, repo: "hexpm", optional: false]}], "hexpm", "da33836dc50f261f306c81444ff6b9e693bc9fe6ecddfdfaa3c90a71cd3257c9"},
   "membrane_rtp_format": {:hex, :membrane_rtp_format, "0.5.0", "1b6930453be836bcc48ff71daeea76ac969cb726c0afda7203ddbbd76ca2db37", [:mix], [{:membrane_core, "~> 0.10.0", [hex: :membrane_core, repo: "hexpm", optional: false]}], "hexpm", "cd4d5c0389b154523d42781e492617e5d02a11af0f11163008f434a9f6179d5a"},
   "membrane_rtp_h264_plugin": {:hex, :membrane_rtp_h264_plugin, "0.13.0", "39c5cbe86471336e8d5cfae0f50d1e1b219b38af908f81d7971efe48c51dfd72", [:mix], [{:bunch, "~> 1.3", [hex: :bunch, repo: "hexpm", optional: false]}, {:membrane_core, "~> 0.10.0", [hex: :membrane_core, repo: "hexpm", optional: false]}, {:membrane_h264_format, "~> 0.3.0", [hex: :membrane_h264_format, repo: "hexpm", optional: false]}, {:membrane_rtp_format, "0.5.0", [hex: :membrane_rtp_format, repo: "hexpm", optional: false]}], "hexpm", "ace81e2783bc63ce54b1375ceb5c2e9c28114f6d03bfad8545b10a88403d1a47"},
   "membrane_rtp_opus_plugin": {:hex, :membrane_rtp_opus_plugin, "0.6.0", "3cb0e7f255567160077a6f8cb5ec4d9193aded19b83561a8a9d9c64a9594f55c", [:mix], [{:membrane_core, "~> 0.10.0", [hex: :membrane_core, repo: "hexpm", optional: false]}, {:membrane_opus_format, "~> 0.3.0", [hex: :membrane_opus_format, repo: "hexpm", optional: false]}, {:membrane_rtp_format, "~> 0.5.0", [hex: :membrane_rtp_format, repo: "hexpm", optional: false]}], "hexpm", "66c54bd9b041dade42c847c0184c633503cf2b829de1ebbf6cacabf4b5e7e9b0"},
-  "membrane_rtp_plugin": {:git, "https://github.com/membraneframework/membrane_rtp_plugin.git", "34739bca4e57e6bd2aed099c8569679f6b6920ad", [branch: "feature/outbound-retransmissions"]},
+  "membrane_rtp_plugin": {:git, "https://github.com/membraneframework/membrane_rtp_plugin.git", "fe83fd8910d4c5ee262906bb297498a7fed0b6d7", [branch: "feature/outbound-retransmissions"]},
```

Comment:
There's one more new commit there - TWCC reset fix https://github.com/membraneframework/membrane_rtp_plugin/commit/015d6f514007e49cf82d929d50e4d96c6f19b73b can you update it before merge?